html/elements/canvas.json

```diff
@@ -0,0 +1,250 @@
+                  "version_added": null
+                },
+                "chrome": {
+                  "version_added": "1.0"
+                },
+                "chrome_android": {
+                  "version_added": null
+                },
+                "edge": {
+                  "version_added": true
+                },
+                "edge_mobile": {
+                  "version_added": true
+                },
+                "firefox": {
+                  "version_added": "1.5",
+                  "notes": [
+                    "Before Firefox 5.0, the canvas width and height were signed integers instead of unsigned integers.",
+                    "Prior to Firefox 6.0, a &lt;canvas&gt; element with a zero width or height would be rendered as if it had default dimensions.",
+                    "Before Firefox 12.0, if JavaScript is disabled, the &lt;canvas&gt; element was being rendered instead of showing the fallback content as per the specification. Since then, the fallback content is rendered instead."
+                  ]
+                },
+                "firefox_android": {
+                  "version_added": "1.5",
+                  "notes": [
+                    "Before Firefox 5.0, the canvas width and height were signed integers instead of unsigned integers.",
+                    "Prior to Firefox 6.0, a &lt;canvas&gt; element with a zero width or height would be rendered as if it had default dimensions.",
+                    "Before Firefox 12.0, if JavaScript is disabled, the &lt;canvas&gt; element was being rendered instead of showing the fallback content as per the specification. Since then, the fallback content is rendered instead."
+                  ]
+                },
+                "ie": {
+                  "version_added": "9.0"
+                },
+                "ie_mobile": {
+                  "version_added": null
+                },
+                "opera": {
+                  "version_added": "9.0"
+                },
+                "opera_android": {
+                  "version_added": false
+                },
+                "safari": {
+                  "version_added": "2.0",
+                  "notes": "Although early versions of Apple's Safari browser don't require the closing tag, the specification indicates that it is required, so you should be sure to include it for broadest compatibility. Those versions of Safari (prior to version 2.0) will render the content of the fallback in addition to the canvas itself unless you use CSS tricks to mask it. Fortunately, users of these versions of Safari are rare nowadays."
+                },
+                "safari_ios": {
+                  "version_added": "1.0"
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          }
+        },
+        "height": {
```

Comment:
Good catch. These are difficult to catch without a simple way to actually generate the table.